clarity/README copy.md

```diff
@@ -0,0 +1,22 @@
+# A Minimalistic NFT Marketplace
+
+Demonstartes a minimalistic NFT marketplace that allows users to list NFTs for sale.
```

Comment:
Typo - "demonstrates"

Also, this file is called `README copy.md`, I think it should just be `README.md`